Tests/SonarScanner.MSBuild.Common.Test/SonarProductTests.cs

```diff
@@ -18,26 +18,22 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
 namespace SonarScanner.MSBuild.Common.Test;
 
 [TestClass]
 public class SonarProductTests
 {
     [DataTestMethod]
-    [DataRow(null, "SonarQube")]
-    [DataRow("https://localhost:9000", "SonarQube")]
-    [DataRow("https://next.sonarqube.com", "SonarQube")]
-    [DataRow("sonarcloud.io", "SonarCloud")]
-    [DataRow("https://sonarcloud.io", "SonarCloud")]
-    [DataRow("https://SonarCloud.io", "SonarCloud")]
-    [DataRow("https://SONARCLOUD.IO.custom.dns.proxy", "SonarCloud")]
-    [DataRow("https://sonarcloud.custom.dns.proxy", "SonarQube")]
-    public void GetSonarProductToLog(string host, string expectedName) =>
-        SonarProduct.GetSonarProductToLog(host).Should().Be(expectedName);
+    [DataRow(null, "SonarQube Server")]
```

Comment:
This should default to `SonarQube Cloud` since we changed the default from `http://localhost:9000` to `https://sonarcloud.io`

Reply:
I did slightly different trickery to avoid the problem

Reply:
Thinking of which... what about dropping it completely and just using `SonarQube` universally everywhere?

Reply:
I think we can drop it. Given the error messages where this is used it doesn't seem to add any value.

Reply:
The PR is recreated from scratch and targets master now, as it has nothing to do with the other changes